### PR TITLE
Added publicPath to output

### PR DESCRIPTION
**Pull request by qasimalyas**
Without this i was finding that webpack wasn't able to find the file which was being dynamically imported which was also reported here https://github.com/webpack/webpack/issues/7502.

Adding this allowed my example to work as expected - makes extra network request to and executes the file.

_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/writers-guide/

### CODE REVIEW

File: src/content/guides/code-splitting.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ __webpack.config.js__
       filename: '[name].bundle.js',
 +     chunkFilename: '[name].bundle.js',
       path: path.resolve(__dirname, 'dist')
+      publicPath: 'dist/'
```
Comment (montogeek) [inline src/content/guides/code-splitting.md]: Does it work removing `/`, I think it is unnecessary.

Comment (qasimalyas) [reply src/content/guides/code-splitting.md]: No, in my example it has to be ``dist/`` for it to work.

**Comment by montogeek** [comment]
@qasimalyas Friendly ping

File: src/content/guides/code-splitting.md (hunk 1 above)
Comment (qasimalyas) [reply src/content/guides/code-splitting.md]: Win 10. Sorry for the late reply.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**